### PR TITLE
add masked card number to response

### PR DESCRIPTION
**Pull request by hakiemaul**
### Problem
We need to return `masked_card_number` during tokenization

### Solution
Add the response in model and example

### CODE REVIEW

File: xendit-android/src/main/java/com/xendit/Models/Authentication.java [hunk 1]
```diff
@@ -23,10 +23,14 @@
     @SerializedName("payer_authentication_url")
     private String payerAuthenticationUrl;
 
+    @SerializedName("masked_card_number")
+    private String maskedCardNumber;
```
Comment (afik) [inline xendit-android/src/main/java/com/xendit/Models/Authentication.java]: I think we're not returning this on authentication right?

Comment (afik) [reply xendit-android/src/main/java/com/xendit/Models/Authentication.java]: oh, token get it from authentication ya.. I forgot, will this work on `createAuthentication`?

Comment (hakiemaul) [reply xendit-android/src/main/java/com/xendit/Models/Authentication.java]: yes this works on authentication only

**Comment by 0505gonzalez** [comment]
Why only on authentication?

**Comment by 0505gonzalez** [comment]
LGTM

**Pull request merged**